testing/test_recwarn.py

```diff
@@ -170,32 +170,6 @@ def f():
                 with pytest.deprecated_call():
                     f()
 
-    def test_deprecated_function_already_called(self, testdir):
```

Comment:
Realized that this test is already covered by `test_deprecated_call_modes` so it was redundant.